<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="UserInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>500</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="AIResponse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sendText">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/sendIcon.png</normaloff>icons/sendIcon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="button_newChat">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>new chat</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/icons/icons/chaticon.png</normaloff>:/icons/icons/chaticon.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_imgGen">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Image generation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>ChatGPT</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/icons/chatgpt.png</normaloff>:/icons/icons/chatgpt.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Google Gemini</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/icons/Gemini.png</normaloff>:/icons/icons/Gemini.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
